### DISCUSSION TITLE
Does Hajk only exist in Swedish?

### DISCUSSION THREAD

**Discussion by AndreasAtakan** [hajkmap/Hajk#943] [Q&A] [answered]
I want to deploy Hajk in English, but I can't find any i18n language-packs or anything like that.

Maybe implement a language-switcher or something.

**Comment by Hallbergs** [comment, marked as answer]
See my answer in #502 

 > The current status is that the core functionality (switching language in the UI or in the map configuration) has been added, and a couple of components have been migrated to use I18n. Unfortunately, the migration has been put on hold due to other priorities. However, i still think it is important that this gets done asap, so that organizations outside Sweden can use Hajk! My guess is that this will get done in q1-q2 2022.
